src/app/jumio/jumio.component.ts

```diff
@@ -0,0 +1,59 @@
+import {Component, OnDestroy, OnInit, Input, Output, EventEmitter} from '@angular/core';
+import {BackendAPIService} from '../backend-api.service';
+import {CryptoService} from '../crypto.service';
+import {AccountService} from '../account.service';
+import {IdentityService} from '../identity.service';
+import {GlobalVarsService} from '../global-vars.service';
+import {environment} from '../../environments/environment';
+import {Router} from '@angular/router';
+import {TextService} from '../text.service';
+import {Network} from '../../types/identity';
+import {RouteNames} from '../app-routing.module';
+
+@Component({
+  selector: 'app-jumio',
+  templateUrl: './jumio.component.html',
+  styleUrls: ['./jumio.component.scss']
+})
+export class JumioComponent implements OnInit, OnDestroy {
+
+  @Input() publicKey: string = "";
+  @Input() showSkip: boolean = true;
+
+  @Output() skipJumio = new EventEmitter();
+
+  constructor(
+    public backendApi: BackendAPIService,
+    public globalVars: GlobalVarsService,
+  ) { }
+
+  ngOnInit(): void {
+  }
+
+  ngOnDestroy(): void {
+  }
+
+  getJumioURL(routeSuffix: string): string {
+    const url = new URL(`${window.location.origin}/${routeSuffix}`)
+    if (this.globalVars.network === Network.testnet) {
+      url.searchParams.append('testnet', 'true');
+    }
+    url.searchParams.append('public_key', this.publicKey);
+    const stateString = btoa(JSON.stringify({
```

Comment:
This appears unused. Remove it?

Reply:
This function is used below in the openJumio function. Is there a specific query param that you think I should remove?

Reply:
I mean the `stateString` is never decoded / used anywhere downstream from what I can see

Reply:
yeah that was just copied from the google flow - will take it out.